src/Sil/Component/Contact/Model/Group.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+/*
+ * Copyright (C) 2015-2018 Libre Informatique
+ *
+ * This file is licenced under the GNU LGPL v3.
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Sil\Component\Contact\Model;
+
+use Doctrine\Common\Collections\Collection;
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @author Romain Sanchez <romain.sanchez@libre-informatique.fr>
+ */
+class Group implements GroupInterface
+{
+    /**
+     * name.
+     *
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * members.
+     *
+     * @var Collection|GroupMemberInterface[]
+     */
+    protected $members;
+
+    /**
+     * parent group.
+     *
+     * @var GroupInterface
+     */
+    protected $parentGroup;
+
+    /**
+     * child groups.
+     *
+     * @var Collection|GroupInterface[]
+     */
+    protected $childGroups;
```

Comment:
Why not only `children` ? (differs from documentation : `childrenGroups` on doc)